src/main/java/seedu/address/model/person/Frequency.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a how often a client's policy is updated in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidFrequency(String)}
+ */
+public class Frequency {
+
+    enum Months {
+        JANUARY, FEBRUARY, MARCH, APRIL, MAY, JUNE, JULY, AUGUST, SEPTEMBER, OCTOBER, NOVEMBER, DECEMBER
```

Comment:
does this line exceed the character limit? if so consider having each month on its own line